run.sh

```diff
@@ -3,6 +3,9 @@
 echo migrate database...
 python /code/crt_portal/manage.py migrate
 
+echo collect static assets...
+python /code/crt_portal/manage.py collectstatic --noinput
```

Comment:
Does this line mean that production and local envs are serving static files in the same way? I was thinking this PR was only changing the production static assets path, but that might be wrong.

Reply:
It will now copy your files locally to /staticfiles, in cloud.gov it will copy to S3, This will make sure that collect static works similarly, in the local environment and doesn't give us surprises in production.

Reply:
Gotcha, thank you!